src/Common/src/Interop/Windows/winhttp/Interop.SafeWinHttpHandle.cs

```diff
@@ -45,12 +64,8 @@ internal sealed class SafeWinHttpHandleWithCallback : SafeWinHttpHandle
             // Add a reference to this object so that the AppDomain doesn't get unloaded before the last WinHttp callback.
             public void AttachCallback()
             {
-                bool success = false;
-                DangerousAddRef(ref success);
-                if (!success)
-                {
-                    throw new InvalidOperationException("Could not increase SafeHandle reference.");
-                }
+                bool ignore = false;
```

Comment:
Thanks for explaining! 
@jkotas : How are we planning to document the new behaviors?